daemons/compute/client-cpp/datamovement.grpc.pb.cc

```diff
@@ -0,0 +1,212 @@
+// Generated by the gRPC C++ plugin.
```

Comment:
Should these 4 generated files be checked in? They can be generated and we've given instruction on how to generate them.

Reply:
You're right they should be removed.